infrastructure/config/config.go

```diff
@@ -577,6 +578,8 @@ func LoadConfig() (*Config, error) {
 		log.Warnf("%s", configFileError)
 	}
 
+	cfg.ActiveNetParams.EnableSanityCheckPruningUTXOSet = cfg.EnableSanityCheckPruningUTXOSet
```

Comment:
This isn't really a NetParam.
I assume you did this so that it's easier to pass the thing into consensus, however, this should be treated the same way we treat isArchivalNode.

Seeing the slippery slope here, I suggest passing all the config to consensus, and let it pick out of it the parts it needs.

Reply:
Theoretically it does "change" consensus behavior, but I get what you're saying.
Assuming that `ActiveNetParams` means network specific configurations, and that `config.Config` or `config.Flags` contains a lot more stuff than consensus will ever need,
maybe we should add another struct in between? `ConsensusConfig` or something. that will contain `ActiveNetParams` and only flags that consensus actually care about.
That way it will also be easier in tests when passing a config to consensus to know that changing something in the config will actually affect consensus and not just be something that seems related to consensus but is actually totally unrelated

Reply:
What you suggest sounds like a good idea, but I suggest we do this in a separate PR.

Meanwhile, treat EnableSanityCheckPruningUTXOSet same way we treat isArchivalNode and pass it separately.